tox.ini

```diff
@@ -72,3 +72,28 @@ deps =
     pylint==2.1.1
 commands =
     python -m pylint --rcfile=.pylintrc -j 0 src/sagemaker
+
+[testenv:sphinx]
+basepython = python3
+changedir = doc
+# Based on: https://github.com/rtfd/readthedocs.org/blob/8f0c78dde5edcc85acf90462a8518735a25482d3/readthedocs/doc_builder/python_environments.py#L263
+install_command = python -m pip install --upgrade -I {packages}
+# Based on: https://github.com/rtfd/readthedocs.org/blob/8f0c78dde5edcc85acf90462a8518735a25482d3/readthedocs/doc_builder/python_environments.py#L280
+deps =
+    Pygments==2.2.0
+    setuptools<40
+    docutils==0.13.1
+    mock==1.0.1
+    pillow==2.6.1
+    alabaster>=0.7,<0.8,!=0.7.5
+    commonmark==0.5.4
+    recommonmark==0.4.0
+    sphinx<1.8
+    sphinx-rtd-theme<0.5
+    readthedocs-sphinx-ext<0.6
+# pip install requirements.txt is separate as RTD does it in separate steps
```

Comment:
There is also doc/requirements.txt used by readthedocs, we probably want to install them as the first step.

Reply:
That is being done right here: https://github.com/aws/sagemaker-python-sdk/pull/653/files/8743f0c62fd94b1e140c1aff4118b96c407943ff#diff-b91f3d5bd63fcd17221b267e851608e8R98

Actually, the requirements.txt is installed right before the sphinx-build and cat doc/conf.py: https://readthedocs.org/projects/sagemaker/builds/8577862/

`/home/docs/checkouts/readthedocs.org/user_builds/sagemaker/envs/v1.18.3.post1/bin/python -m pip install --exists-action=w --cache-dir /home/docs/checkouts/readthedocs.org/user_builds/sagemaker/.cache/pip -r doc/requirements.txt`

Reply:
oh, right, missed the changedir logic above.